ocaml/middle_end/semantics_of_primitives.ml

```diff
@@ -160,7 +160,7 @@ let for_primitive (prim : Clambda_primitives.primitive) =
   | Pget_header _ -> No_effects, No_coeffects
   | Pdls_get ->
       (* only read *)
-      No_effects, No_coeffects
```

Comment:
Why was this changed?

Reply:
Hmm, well I think this operation has coeffects - it's a memory load?

Reply:
I checked upstream and it's `No_coeffects` there; I think it's correct because while it's a memory load, it's never to a location that is written to.